.flake8

```diff
@@ -2,3 +2,6 @@
 ignore = E203, E266, W503, E402
 max-line-length = 88
 exclude = tests/yatai/proto, proto, _version.py
+per-file-ignores =
+    docker/manager.py:E501
+    docker/utils.py:W605, E501
```

Comment:
W605: Are you encountering escape issue in long strings?
You may need r"\" or r''' \ '''

Reply:
This is limited to how `cerberus` parse regex. It only accept regex that encapsulate between single quotes, eg. 

```yaml

field:
  regex: '(\w+)\,{1}(\d+)'
```